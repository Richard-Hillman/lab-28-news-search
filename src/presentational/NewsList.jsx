import React from 'react';
import propTypes from 'prop-types';

const NewsList = ({ searchResults }) => (
  // eslint-disable-next-line max-len
  <div style={{ backgroundColor: 'black', width: '50vh', height: '70vw', color: 'white' }}>
    
    <h1> Results </h1>
  
    {
      searchResults.map((result) => (
        <p key="i">{ result.author }</p>
      )
      )};


  </div>
);

NewsList.propTypes = {
  searchResults: propTypes.string.isRequired,
 
};

export default NewsList;
