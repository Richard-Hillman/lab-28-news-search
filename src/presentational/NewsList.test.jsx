import React from 'react';
import { render, cleanup } from '@testing-library/react';
import NewsList from './NewsList';

describe('NewsList component', () => {
  afterEach(() => cleanup());
  it('renders NewsList', () => {
    const { asFragment } = render(<NewsList
      searchResults= {[]} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
