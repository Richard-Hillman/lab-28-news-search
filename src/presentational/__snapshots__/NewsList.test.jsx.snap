// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsList component renders NewsList 1`] = `
<DocumentFragment>
  <div
    style="background-color: black; width: 50vh; height: 70vw; color: white;"
  >
    <h1>
       Results 
    </h1>
  </div>
</DocumentFragment>
`;
